### PR TITLE
Make helpers::fabric::Command public again.

### PR DESCRIPTION
**Pull request by alekseysidorov**

### CODE REVIEW

File: exonum/src/helpers/fabric/internal.rs [hunk 1]
```diff
@@ -29,9 +29,13 @@ pub enum Feedback {
 
 /// `Command` trait is used to create sub-command for `NodeBuilder`.
 pub trait Command {
+    /// List of command arguments.
     fn args(&self) -> Vec<Argument>;
+    /// Command name.
     fn name(&self) -> CommandName;
+    /// Short information about the command.
     fn about(&self) -> &str;
+    #[doc(hidden)]
```
Comment (vitvakatu) [inline exonum/src/helpers/fabric/internal.rs]: Any reason to do so?

**Comment by DarkEld3r** [comment]
Totally forgot about the changelog: it should be updated.

File: exonum/src/helpers/fabric/internal.rs [hunk 2]
```diff
@@ -29,9 +29,19 @@ pub enum Feedback {
 
 /// `Command` trait is used to create sub-command for `NodeBuilder`.
 pub trait Command {
+    /// List of command arguments.
     fn args(&self) -> Vec<Argument>;
+    /// Command name.
     fn name(&self) -> CommandName;
+    /// Short information about the command.
     fn about(&self) -> &str;
+    /// Performs command handling code.
```
Comment (popzxc) [inline exonum/src/helpers/fabric/internal.rs]: I'd suggest `Executes the command processing code.` or something like that.

File: exonum/src/helpers/fabric/internal.rs [hunk 3]
```diff
@@ -29,9 +29,19 @@ pub enum Feedback {
...
+    ///
+    /// # Arguments:
+    ///
+    /// * `commands` - contains list of commands that can be performed on the node.
```
Comment (popzxc) [inline exonum/src/helpers/fabric/internal.rs]: `list of commands` sounds weird when argument type is a `HashMap`.

File: exonum/src/helpers/fabric/internal.rs [hunk 4]
```diff
@@ -29,9 +29,19 @@ pub enum Feedback {
...
+    /// * `context` - contains command parameters.
```
Comment (popzxc) [inline exonum/src/helpers/fabric/internal.rs]: Is it really `command parameters`? I thought it's more like global execution context.

File: CHANGELOG.md [hunk 5]
```diff
@@ -9,6 +9,13 @@ The project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
 #### exonum
 
+- `Command` trait in `helpers::fabric` module became public. (#778)
+
+  Migration path:
+
+  If you overrides `ServiceFactory::command` method and matches command name,
```
Comment (popzxc) [inline CHANGELOG.md]: Override, do a `match` on a command name.

**Pull request merged**